uninstall.php

```diff
@@ -1,32 +1,36 @@
 <?php
-
 /**
- * WooCommerce Uninstall
+ * Rt newrelic browser Uninstall
  *
- * Uninstalling WooCommerce deletes user roles, options, tables, and pages.
+ * Uninstalling rt-newrelic-browser deletes options.
  *
- * @author 		WooThemes
- * @category 	Core
- * @package 	WooCommerce/Uninstaller
- * @version     2.1.0
+ * @author      rtCamp
+ * @category    Core
+ * @package     NewRelicBrowser
+ * @version     1.0.5
  */
+
 if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
 	exit();
 }
+
 global $wpdb;
```

Comment:
Remove this global variable because its not used.

Reply:
i solved this issue. please review changes.